src/components/Navbar.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+export default function navbar() {
+  return (
+    <div>navbar</div>
+  );
+}
```

Comment:
Please implement the navbar component having Links to display Home and About Page